PilotDataPlatform/bff-web
# PILOT-1412: Modify response of querying file attributes

## Summary

- Modified `/file/manifest/query` endpoint to return all values of optional attributes that are not explicitly assigned by a user (`null` values).

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-1412

## Type of Change

Please delete options that are not relevant.


## Testing

Are there any new or updated tests to validate the changes?


## Test Directions

1. Create file attribute template with one attribute name that is set to **optional**.
2. Attach attribute template to any file, without selecting `null` for the given attribute name.
3. Check Portal `File Attributes` tab for the file, where the following attribute should be displayed: `(attribute_name: null)`
